project/.pre-commit-config.yaml

```diff
@@ -1,47 +1,32 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
+# see https://github.com/VOLTTRON/volttron-core/
 repos:
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.0.1
+  rev: v4.1.0
   hooks:
-  - id: check-added-large-files
-  - id: check-toml
   - id: check-yaml
+  - id: check-json
+  - id: check-toml
+  - id: check-xml
+  - id: forbid-new-submodules
   - id: end-of-file-fixer
   - id: trailing-whitespace
-- repo: https://github.com/psf/black
-  rev: 22.3.0
-  hooks:
-  - id: black
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v0.910-1
-  hooks:
-  - id: mypy
-    exclude: ^(docs/|example-plugin/|tests/fixtures)
-- repo: local
-  hooks:
-  - id: pytest
-    name: Check pytest unit tests pass
-    entry: poetry run test
-    pass_filenames: false
-    language: system
-    types: [python]
-- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
-  rev: v2.2.0
-  hooks:
-  - id: pretty-format-toml
-    args: [--autofix]
-  - id: pretty-format-yaml
-    args: [--autofix]
-    exclude: .copier-answer.yml
-    # based on
-    # https://gitlab.com/smop/pre-commit-hooks/-/blob/master/.pre-commit-hooks.yaml
-- repo: local
+  - id: check-merge-conflict
+  - id: no-commit-to-branch  # blocks main commits.  To bypass do git commit --allow-empty
+  - id: pretty-format-json
+
+#- repo: https://github.com/pre-commit/mirrors-autopep8
+#  rev: v1.6.0
+#  hooks:
+#    - id: autopep8
+
+- repo: https://github.com/craig8/mirrors-yapf
```

Comment:
Let's include a comment about what YAPF is about. I don't think too many developers are aware of this hook. Include a comment that looks like:

```yaml
#  For more information about YAPF, see https://github.com/google/yapf
```